capstone/src/main/java/com/google/sps/servlets/DashboardHandlerServlet.java

```diff
@@ -15,6 +24,20 @@
 @WebServlet("/dashboard-handler")
```

Comment:
Servlets dont need "-handler" on them since, by definition unless the request gives like 404 or something, you are automatically asking a handler to handle your request.

Reply:
Got it.

Reply:
Removed